src/App.test.tsx

```diff
@@ -2,8 +2,6 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('renders app.js', () => {
   const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
```

Comment:
Looks like we've really just deleted the test.
Can we instead test for an element that does still exist?

Reply:
Lol! I've made a test to check the url for the curiosity page.